src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.pkgproj

```diff
@@ -5,12 +5,14 @@
     <Version>1.1.0</Version>
     <SkipPackageFileCheck>true</SkipPackageFileCheck>
     <SkipValidatePackage>true</SkipValidatePackage>
-    <PackagePlatforms>x64;x86;arm64;arm</PackagePlatforms>
+    <PackagePlatforms>x64;x86;arm64;arm;</PackagePlatforms>
     <OutputPath>$(PackagesOutputPath)</OutputPath>
     <IncludeRuntimeJson>true</IncludeRuntimeJson>
   </PropertyGroup>
   <ItemGroup>
-    <!-- Declare a runtime dependency on the win8-arm CoreCLR built using the TFS builds -->
+    <!-- ARM32_TODO: Clean this up.
```

Comment:
@weshaggard I don't intend to remove this dependency, in the immediate future, while still producing the packages as part of the WinArm32 build. Once we have validated the E2E for UWP, with Reverse Forwarders, I will clean this up.

Do you see any concerns with this approach?